netlify/functions/deploy-succeeded.js

```diff
@@ -1,5 +1,3 @@
-const fetch = require('node-fetch');
```

Comment:
ok, this wasn't even in package.json? 😄

Reply:
Nope. 🙈 
We relied on a transitive dependency. 😆